mantidproject/mantid
# Sans v2 continuous plotting

This work adds an option in the SANS GUI to continuously plot the latest reduction results. 

**To test:**
- Get the loqdemo data from the SANS example data. This can be found http://download.mantidproject.org/
- Open the sans V2 Gui
- Load maskfile.txt as the user file
- Load batch_file.csv as the batch file. 
- Run a reduction with the plot results box ticked the data should be plotted at the end. This tick box is on the bottom right of the GUI window.


Fixes #21222 . 

**Release Notes** 
Added to release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
